### ISSUE TITLE
AAF for binary labels

### ISSUE THREAD

**Issue by arc144** [twke18/Adaptive_Affinity_Fields#10]
Hi, in the original implementation there are multiple labels (background, label 1, labels 2, etc..). In my application I have only one class besides background. Therefore I use sigmoid and have logits in the shape (B, H, W, 1). Should I expand my labels and predictions to (B, H, W, 2) where labels/pred[..., 1] = 1 - pred[..., 0]?

Or can I just compute for a single channel since background and foreground are complementary for the binary case?

Thank you,
Kind regards

**Comment by twke18** [comment]
Yes, you can just use a single channel in the case of binary classification.
